### DISCUSSION TITLE
Is it possible to add GPT-4 into the model list?

### DISCUSSION THREAD

**Discussion by Space-f-Monkey** [yihong0618/bilingual_book_maker#212] [Ideas]
I was looking at the code, but my coding knowledge is limited so I don't want to mess things up too much.
But as far as I can tell, you only have to put "gpt4" into cli.py line 223 for bilingual_book_maker to translating books with GPT-4?

**Comment by yihong0618** [comment]
for now I do not have GPT-4 api, I think we can add it.
PR welcome

**Comment by Jackie00765** [comment]
Is the gpt-4 supported now?

**Reply by Space-f-Monkey** [reply]
I haven't get whitelisted, so I don't know.
It shouldn't be hard though. If you get GPT-4, feel free to try it.

**Comment by Space-f-Monkey** [comment]
Will you update with Bard API since it is cheaper than GPT now?
Bard's accuracy is kind of more closer to GPT-4, but they can always change pricing laster.
